pkg/network/ebpf/c/prebuilt/tracer.c

```diff
@@ -460,6 +457,22 @@ int kprobe__udp_recvmsg(struct pt_regs *ctx) {
 
 SEC("kprobe/udpv6_recvmsg")
 int kprobe__udpv6_recvmsg(struct pt_regs *ctx) {
+    struct sock *sk = (struct sock *)PT_REGS_PARM1(ctx);
+    struct msghdr *msg = (struct msghdr *)PT_REGS_PARM2(ctx);
+    int flags = (int)PT_REGS_PARM5(ctx);
+    handle_udp_recvmsg(sk, msg, flags, udp_recv_sock);
+}
+
+SEC("kprobe/udp_recvmsg/pre_4_7_0")
```

Comment:
You don't need to have a different section name now, since only the function name is used for program identification.

```suggestion
SEC("kprobe/udp_recvmsg")
```

Reply:
This means you don't need the `MatchFuncName` later when configuring the manager